modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java

```diff
@@ -1322,7 +1324,24 @@ private MockLiferayPortletRenderRequest _getMockLiferayPortletRenderRequest(
 		throws Exception {
 
 		MockLiferayPortletRenderRequest mockLiferayPortletRenderRequest =
-			new MockLiferayPortletRenderRequest();
+			new MockLiferayPortletRenderRequest() {
+
+				@Override
+				public PortletPreferences getPreferences() {
```

Comment:
Could we move this code to the class MockLiferayPortletRenderRequest? adding a setter, do we